.github/workflows/push-image.yml

```diff
@@ -3,184 +3,126 @@ name: Push Stack Image
+          # sudo skopeo copy "oci-archive:${GITHUB_WORKSPACE}/${{ matrix.oci_image }}.oci" "docker://gcr.io/${GCR_PROJECT}/${{ matrix.oci_image }}-${{ steps.registry-repo.outputs.name }}:${{ steps.event.outputs.tag }}"
+          # sudo skopeo copy "oci-archive:${GITHUB_WORKSPACE}/${{ matrix.oci_image }}.oci" "docker://gcr.io/${GCR_PROJECT}/${{ matrix.oci_image }}-${{ steps.registry-repo.outputs.name }}:latest"
+
+          # If the repository name contains 'bionic', let's push it to legacy image locations as well:
+          #    paketobuildpacks/{build/run}:{version}-{variant}
+          #    paketobuildpacks/{build/run}:{version}-{variant}-cnb
+          #    paketobuildpacks/{build/run}:{variant}-cnb
+          #    paketobuildpacks/{build/run}:{variant}
+          registry_repo="${{ steps.registry-repo.outputs.name }}"
+          if [[ ${registry_repo} == "bionic"-* ]];
+            then
+            # Strip the final part from a repo name after the `-`
+            # bionic-tiny --> tiny
+            variant="${registry_repo#bionic-}"
+
+            sudo skopeo copy "oci-archive:${GITHUB_WORKSPACE}/${{ matrix.oci_image }}.oci" "docker://${DOCKERHUB_ORG}/${{ matrix.oci_image }}:${{ steps.event.outputs.tag }}-${variant}"
+            sudo skopeo copy "oci-archive:${GITHUB_WORKSPACE}/${{ matrix.oci_image }}.oci" "docker://${DOCKERHUB_ORG}/${{ matrix.oci_image }}:${{ steps.event.outputs.tag }}-${variant}-cnb"
+            sudo skopeo copy "oci-archive:${GITHUB_WORKSPACE}/${{ matrix.oci_image }}.oci" "docker://${DOCKERHUB_ORG}/${{ matrix.oci_image }}:${variant}-cnb"
+            sudo skopeo copy "oci-archive:${GITHUB_WORKSPACE}/${{ matrix.oci_image }}.oci" "docker://${DOCKERHUB_ORG}/${{ matrix.oci_image }}:${variant}"
+
+            # uncomment below line when gcr is back
+            # sudo skopeo copy "docker://${DOCKERHUB_ORG}/${{ matrix.oci_image }}:${variant}-cnb" "docker://gcr.io/${GCR_PROJECT}/${{ matrix.oci_image }}:${variant}-cnb"
+          fi
 
   failure:
     name: Alert on Failure
     runs-on: ubuntu-22.04
     needs: [push]
     if: ${{ always() && needs.push.result == 'failure' }}
     steps:
-    - name: File Failure Alert Issue
-      uses: paketo-buildpacks/github-config/actions/issue/file@main
-      with:
-        token: ${{ secrets.GITHUB_TOKEN }}
-        repo: ${{ github.repository }}
-        label: "failure:push"
-        comment_if_exists: true
-        issue_title: "Failure: Push Image workflow"
-        issue_body: |
-          Push Image workflow [failed](https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}).
-          Please take a look to ensure CVE patches can be released. (cc @paketo-buildpacks/stacks-maintainers).
-        comment_body: |
-           Another failure occurred: https://github.com/${{github.repository}}/actions/runs/${{github.run_id}}
+      - name: File Failure Alert Issue
```

Comment:
same question about all these space additions?

Reply:
The same applies here too. I'm happy to revert the changes or raise a question on the next paketo working group or both :)